shellcheck/shellcheck.yml.yaml

```diff
@@ -0,0 +1,15 @@
+files:
+  - repo: console-login-helper-messages
+    path: .github/workflows/shellcheck.yml
+
+  - repo: coreos-installer
```

Comment:
I'll note that this will validate not only the dracut module, but also the test scripts.  This may or may not turn out to be a good idea.

Reply:
I'm doing a pass for all our repos before merging this one so that once we merge it we should only get "new" errors.